README.md

```diff
@@ -148,6 +150,7 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 
 ### Linux Evidence Collection
 
+* [Acquire](https://github.com/fox-it/acquire) - Acquire is a tool to quickly gather forensic artifacts from disk images or a live system into a lightweight container. This makes Acquire an excellent tool to, among others, speedup the process of digital forensic triage. It uses [Dissect](https://github.com/fox-it/dissect) to gather that information from the raw disk, if possible.
```

Comment:
you can add acquire to only 1 section, not to 3

Reply:
Changed!